addons/mail/static/src/core/common/thread_model.js

```diff
@@ -249,6 +249,15 @@ export class Thread extends Record {
      *  @type {integer|undefined}
      */
     pid;
+    /**@type {Boolean} */
```

Comment:
I think we can omit the docstring, we use a script that generates the type defs and it can deduce the type form the attr first param.